scipy/special/tests/test_wrightomega.py

```diff
@@ -52,7 +52,7 @@ def test_wrightomega_singular():
     for p in pts:
         res = sc.wrightomega(p)
         assert_equal(res, -1.0)
-        assert_(np.signbit(res.imag) == False)
+        assert_(np.signbit(res.imag) is False)
```

Comment:
We can't use `is` here.  The return value of `np.signbit(res.imag)` is an instance of `np.bool_`.  It is equal to `False`, but it is a different object.  How about using
```
        assert_(not np.signbit(res.imag))
```
instead?  (Or just go back to `==`.)